src/frontend/src/pages/index.js

```diff
@@ -90,17 +91,20 @@ export default function IndexPage() {
   }
 
   const getPostsCount = useCallback(async () => {
-    try {
-      const res = await fetch(`${telescopeUrl}/posts`, { method: 'HEAD' });
-      if (!res.ok) {
-        throw new Error(res.statusText);
+    if (update) {
+      try {
+        const res = await fetch(`${telescopeUrl}/posts`, { method: 'HEAD' });
```

Comment:
So how does this update logic work?  After the first load, you want to keep checking the number of posts, and if it changes, we know we have new ones?

If so, I wonder if "update" is the right name.  `shouldCheckForNewPost` or something (shorter) but better documenting?